src/mlmd/EdgeLine.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+// @ts-ignore
+import LineChart from 'react-svg-line-chart';
+
+interface EdgeLineProps {
+  height: number;
+  width: number;
+  y1: number;
+  y4: number;
+}
+
+export const EdgeLine: React.FC<EdgeLineProps> = (props) => {
+  const {height, width, y1, y4} = props;
+
+  return <LineChart
+    data={[
+      {x: 0, y: y1},
+      {x: 30, y: y1},
+      {x: width - 30, y: y4},
+      {x: width     , y: y4},
+    ]}
+    areaVisible={false}
+    axisVisible={false}
+    gridVisible={false}
+    labelsVisible={false}
+    pathColor={'#BDC1C6'}
```

Comment:
Let's have the `edgeColor` be a css passed color. 
```suggestion
    pathColor={'currentColor'}
```

Now you can simply assign a color on the html node, and the edge color should automatically update, all through css!

Reply:
That doesn't compute to the same color. https://screenshot.googleplex.com/qpLgGEoSFuB

Reply:
Well, you also need to update the CSS for `LineChart` to now contain this color. Basically moving the coloring onus to CSS instead of JS.

Reply:
Not sure I get how that would work - can you explain which files to modify?

Also I'm not sure how readable it is to have this component take in props about everything except the color which would be coming from a different file (I assume). What's the optimization here?

Reply:
Take a look at this:
![image](https://user-images.githubusercontent.com/5303018/74789882-de6e4100-526a-11ea-8043-ef90189175ca.png)

I personally feel that colors shouldn't be props, all aesthetic changes should be done within CSS. This essentially allows a user to CSS theme their component however they feel, rather than having to programmatically change the color props of individual components. With this change, you can simply do:

```css
.edgeCanvas {color: red} /* that's it!* /
.edgeCanvas:nth-of-type(5) {color: darkRed}
```

Reply:
That won't work though. The CSS class is `edgeCanvas_fbvqlg9` and generated by typestyle.